Python/rerf/RerF.py

```diff
@@ -83,6 +91,14 @@ def fastRerF(
     forestClass.setParameter("numTreesInForest", trees)
     forestClass.setParameter("minParent", minParent)
 
+    if binMin > 0:
+        forestClass.setParameter("binMin", binMin)
+        forestClass.setParameter("binSize", binSize)
+        if not(forestType == "rerf" or forestType == "rfBase"):
```

Comment:
maybe move this above the set parameters?

Reply:
So, that's fine but I wonder if we should have the user facing parameter be number of bins? Basically which do we want to keep constant?